### DISCUSSION TITLE
Graphql beta/ga + manually updating data

### DISCUSSION THREAD

**Discussion by alexanderjophus** [PokeAPI/pokeapi#990] [General]
Hi,

Firstly, thank you for maintaining this, it's been amazing to use. I'm trying to migrate from using REST to graphql, for the purposes of [my website](https://pokedex.alexanderjophus.dev/), for a few reasons
- Less data over the wire that I need
- Easier way to get specifically which regional variant is in a specific pokedex
- Less requests in general

It's been really useful and well thought out. My only issue as I understand is it's manually updated to include new data - the REST api has data the graphql one does not have. Is this something I/we the community can help with?

Also, is there a roadmap for when graphql goes out of beta? Does the manual data go hand to hand with this?

**Comment by Naramsim** [comment]
Hi I'm developing tight now the solution to this. 

Currently I'm upgrading the Debian machine and all the systems are down. 

Later I'll put in place the cicd connection between GitHub and gcp for updating the data automatically . It works I already tested it. 

We will not get out of beta though

**Reply by alexanderjophus** [reply]
Thank you! Can confirm, I just tried blueberry and it works. Thanks for your hard work on this project!
